ncm-ceph/src/test/perl/configure.t

```diff
@@ -0,0 +1,37 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Quattor qw(configure);
+use Test::Quattor::Object;
+use NCM::Component::ceph;
+use cfgdata;
+use osddata;
+use clusterdata;
+
+my $cfg = get_config_for_profile("configure");
+
+my $cmp = NCM::Component::ceph->new('ceph');
+isa_ok($cmp, 'NCM::Component::ceph', 'got ncm-ceph instance');
```

Comment:
also add `isa_ok($cmp, 'NCM::Component::Ceph::Luminous', 'got ncm-ceph Luminous instance');`

Reply:
it isn't. ``ncm-ceph Luminous instance isn't a 'NCM::Component::Ceph::Luminous' it's a 'NCM::Component::ceph'``

Reply:
actually it only becomes 'NCM::Component::Ceph::Luminous' after configure, see https://github.com/quattor/ncm-ncd/pull/124/files#diff-839a15e492e80021f8faf6acd665431bR61